helpers/db.js

```diff
@@ -77,6 +77,11 @@ const drop = (tables, done) => {
   );
 };
 
+connect(MODE_TEST, () => {
```

Comment:
This doesn't make sense...you should be wrapping all the code you call in this, not calling it in the helper file and no-oping the callback

Reply:
tbh I don't even know why I put that there.